test_app/CMakeLists.txt

```diff
@@ -5,7 +5,7 @@ cmake_minimum_required(VERSION 3.5)
 set(EXTRA_COMPONENT_DIRS ../libsodium ../expat)
 
 # Set the components to include the tests for.
-set(TEST_COMPONENTS libsodium expat CACHE STRING "List of components to test")
+set(TEST_COMPONENTS libsodium expat esp_encrypted_img CACHE STRING "List of components to test")
```

Comment:
Why this change is required? Can we remove it?

Reply:
I'd vote for keeping it here. At least we are sure that the component compiles fine on all targets and supported IDF version.
Later we can run those tests too.